### ISSUE TITLE
scylla-housekeeping: fail to create `/etc/scylla.d/housekeeping.uuid` for permission denied

### ISSUE THREAD

**Issue by kongove** [scylladb/scylla#2009]
*Installation details*
Scylla version (or git commit hash):  1.6.rc0-20170112.7b40e19 (built by myself)
Cluster size: 3
OS (RHEL/CentOS/Ubuntu/AWS AMI):   ami-c6c122d0 (built by myself)

Commit 05f2bf5bd599b033d34e9431da6828066a1ff44a introduced a new problem, it's failed to generate housekeeping.uuid file.

```
Jan 12 05:20:01 ip-10-166-193-103 sshd[2066]: pam_unix(sshd:session): session opened for user centos by (uid=0)
Jan 12 05:20:01 ip-10-166-193-103 scylla-housekeeping[2175]: Traceback (most recent call last):
Jan 12 05:20:01 ip-10-166-193-103 scylla-1.6.rc0-20170112.7b40e19
housekeeping[2175]: File "/usr/lib/scylla/scylla-housekeeping", line 125, 
in <module>
Jan 12 05:20:01 ip-10-166-193-103 scylla-housekeeping[2175]: create_uuid_file(args.uuid_file)
Jan 12 05:20:01 ip-10-166-193-103 scylla-housekeeping[2175]: File "/usr/lib/scylla/scylla-housekeeping", line 69, i
n create_uuid_file
Jan 12 05:20:01 ip-10-166-193-103 scylla-housekeeping[2175]: with open(args.uuid_file, 'w') as myfile:
Jan 12 05:20:01 ip-10-166-193-103 scylla-housekeeping[2175]: IOError: [Errno 13] Permission denied: '/etc/scylla.d/
housekeeping.uuid'
Jan 12 05:20:01 ip-10-166-193-103 systemd[1]: scylla-housekeeping.service: main process exited, code=exited, status
=1/FAILURE
Jan 12 05:20:01 ip-10-166-193-103 systemd[1]: Unit scylla-housekeeping.service entered failed state.
Jan 12 05:20:01 ip-10-166-193-103 systemd[1]: scylla-housekeeping.service failed.
```

**Comment by syuu1228** [comment]
This is because we changed housekeeping user from root to scylla.
But I don't feel making /etc/scylla.d writable from scylla user, probably it's not secure.
Can we move the file somewhere on /var/lib/scylla?

**Comment by avikivity** [comment]
`/var/lib/scylla` is not a good place, the user may want to mount a partition over it, hiding existing files.

Maybe `/var/something/else`.

**Comment by syuu1228** [comment]
Then maybe we need to create /var/lib/scylla-housekeeping.

**Comment by kongove** [comment]
Verified that Takuya's patch (bee7f54) fixed Issue #2004 and Issue #2009, Unique uuid file can be generated for each AMI instance.
